res/database/Default/user/scr0llbaer_news.xml

```diff
@@ -173,166 +196,207 @@
 				<squirrel_0>
 					<de>Eichhörnchen</de> <!-- alle nur Maskulinum und Neutrum! -->
 					<en>squirrel</en>
+					<pl>wiewiórka</pl>
 				</squirrel_0>
 				<squirrel_1>
 					<de>Streifenhörnchen</de>
 					<en>chipmunk</en>
+					<pl>wiewiórka</pl>
```

Comment:
same as squirrel on purpose?

Reply:
This file will be in push with xml.
There is only one term for squirrel in Polish, not like in German and English.